contest/templates/contest/message.html

```diff
@@ -0,0 +1,35 @@
+{% if messages %}
+    {% for message in messages %}
+    {% if message.tags == 'info' %}
+    <div class="alert alert-info alert-dismissible" role="alert">
+      <button type="button" class="close" data-dismiss="alert" aria-label="Close">
+        <span aria-hidden="true">&times;</span></button>
+      <a>{{ message }}</a>
+    </div>
+    {% elif message.tags == 'success' %}
+    <div class="alert alert-success alert-dismissible" role="alert">
+      <button type="button" class="close" data-dismiss="alert" aria-label="Close">
+        <span aria-hidden="true">&times;</span></button>
+      <a>{{ message }}</a>
+    </div>
+    {% elif message.tags == 'warning' %}
+    <div class="alert alert-warning alert-dismissible" role="alert">
+      <button type="button" class="close" data-dismiss="alert" aria-label="Close">
+        <span aria-hidden="true">&times;</span></button>
+      <a>{{ message }}</a>
+    </div>
+    {% elif message.tags == 'error' %}
+    <div class="alert alert-danger alert-dismissible" role="alert">
+      <button type="button" class="close" data-dismiss="alert" aria-label="Close">
+        <span aria-hidden="true">&times;</span></button>
+      <a>{{ message }}</a>
+    </div>
+    {% else %}
+    <div class="alert alert-info alert-dismissible" role="alert">
```

Comment:
Since you use info as default alert type, is it ok to remove first if statement?
(i.e. {% if message.tag=='info' %})